builder.md

```diff
@@ -46,25 +51,29 @@ Scheme files have the following structure:
     scheme: "Scheme Name"
     author: "Scheme Author"
     description: "a short description of the scheme"
-    base00: "000000"
-    base01: "111111"
-    base02: "222222"
-    base03: "333333"
-    base04: "444444"
-    base05: "555555"
-    base06: "666666"
-    base07: "777777"
-    base08: "888888"
-    base09: "999999"
-    base0A: "aaaaaa"
-    base0B: "bbbbbb"
-    base0C: "cccccc"
-    base0D: "dddddd"
-    base0E: "eeeeee"
-    base0F: "ffffff"
-
-- Hexadecimal color values may optionally be preceded by a "#".
+    style: base16
```

Comment:
I'd prefer `system` since style could be confused for `scheme` which could be confused for name... where-as I thnk system is clearer that it's not the name.

Reply:
But I'd also point out you're still thinking that "builders own the systems" rather than "systems guide the builders"... having a system explicitly specify which system it is somewhere (if it's unclear from context) - _but I'm not sure that is a thing that the builder should dictate to the system._

Rather the system should tell the builder where it might learn about that information.

Reply:
I think longer-term we shouldn't even assume there is a YAML file with keys at all... (much less assume we know the meanings of all those keys universally)

Reply:
`system` seems reasonable - I can change that.

I'm currently focusing on "builders implement the systems" because that's how it's been and I'm still trying to avoid rocking the boat too much.

This change is primarily assuming that we're focusing on palate-based systems because that's all we really have (for now).

Reply:
> This change is primarily assuming that we're focusing on palate-based systems 

I hear you but I don't see how `system:` mandates what kind of systems...  it seems to me that potentially any YAML based system with a `system:` key technically would work. :-)